package.json

```diff
@@ -86,13 +86,13 @@
     "@commitlint/config-conventional": "^19.2.2",
     "@openapitools/openapi-generator-cli": "^2.13.4",
     "@schematics/angular": "18.1.2",
-    "@storybook/addon-essentials": "8.2.6",
```

Comment:
Bitte ALLE "storybook" und "msw" Einträge löschen - brauchen wir generell nicht in den Produkten